### DISCUSSION TITLE
Cluster Tools tip in cluster dashboard

### DISCUSSION THREAD

**Discussion by segevfiner** [rancher-sandbox/rancher-desktop#5495] [Q&A] [answered]
Does this tip mean anything? Or is it just a leftover due to reusing the Rancher cluster dashboard for this UI? As in, is there any feature to install stuff like logging into the cluster, like in the full Rancher, in Rancher Desktop?

<img width="554" alt="image" src="https://github.com/rancher-sandbox/rancher-desktop/assets/24731903/862f571b-ae8d-4f33-afca-3b93937568e7">

**Comment by jandubois** [comment, marked as answer]
Please ignore it; it should not be displayed (see #4168).
